selfservice/flow/verification/handler.go

```diff
@@ -248,15 +249,51 @@ func (h *Handler) fetch(w http.ResponseWriter, r *http.Request, _ httprouter.Par
 
 	if req.ExpiresAt.Before(time.Now().UTC()) {
 		if req.Type == flow.TypeBrowser {
+			reason := "The verification flow has expired. Redirect the user to the verification flow init endpoint to initialize a new verification flow."
```

Comment:
you can also use

```suggestion
			reportErr := x.ErrGone.WithReason("The verification flow has expired. Redirect the user to the verification flow init endpoint to initialize a new verification flow.")
```
and later do 
```go
			h.d.Writer().WriteError(w, r, errors.WithStack(reportErr.
				WithDetail("redirect_to", RequestUrl)))
			return
```